openwisp_controller/config/base/config.py

```diff
@@ -241,6 +241,7 @@ def manage_vpn_clients(cls, action, instance, pk_set, **kwargs):
 
     @classmethod
     def clean_templates_org(cls, action, instance, pk_set, **kwargs):
+        templates = cls._get_templates_from_pk_set(pk_set)
```

Comment:
I don't understand, this looks repeated (after 3 lines we call this again), why is that?
Looks like a mistake to me.